Source/WebKit/NetworkProcess/ServiceWorker/ServiceWorkerDownloadTask.cpp

```diff
@@ -86,7 +86,7 @@ template<typename Message> bool ServiceWorkerDownloadTask::sendToServiceWorker(M
     if (!m_serviceWorkerConnection)
         return false;
 
-    return m_serviceWorkerConnection->ipcConnection().send(std::forward<Message>(message), 0) == IPC::Error::NoError;
+    return !m_serviceWorkerConnection->ipcConnection().send(std::forward<Message>(message), 0);
```

Comment:
265125@main from a month ago just changed this from `send(...)` to `send(...) == IPC::Error::NoError` and now this patch changes it to `!send(...)` which seems to invert the previous meaning but maybe I am misunderstanding.

Is there a downside to leaving the explicit comparison against `IPC::Error::NoError`?

Reply:
Oh it is because prior to `265125@main` `send()` returned true on success, after that change it returned an IPC::Error on success, with 0 representing success (that is where the inversion was introduced)